src/constants/configs.ts

```diff
@@ -1,6 +1,7 @@
 export const languageMappingsWithText: {[key: string]: string} = {
     c: "printf(\"{selectedSnippet}: %d\", {selectedSnippet});",
     cpp: "cout << \"{selectedSnippet}: \" << {selectedSnippet};",
+    go: "fmt.Printf(\"{selectedSnippet}: %d\", {selectedSnippet})",
```

Comment:
Isn't `%d` is format specifier that replaces integer variables with their values?

I'd prefer to match with anything that is selected. How is the following recommendation working for you, what do yo think?
```suggestion
    go: "fmt.Printf(\"{selectedSnippet}: \", {selectedSnippet})",
```

Reply:
You're right! LGTM.